src/routes/auth.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const router = express.Router();
+
+const { login, register, getUserById } = require('../controllers/auth');
```

Comment:
[More git troubles]
See your PR for PRACT-30 if you haven't already. You really can't merge a PR that references code that's not on the PRs branch because it would break the repo for everyone.

In this case you may need to merge all the auth/user changes into a single branch and push a new PR for that branch. We can do it together if you'd like help.

Reply:
I merged both branches into one and closed PR PRACT-30.